docs/source/release/v3.11.0/framework.rst

```diff
@@ -11,10 +11,9 @@ Algorithms
 New
 ###
 
-
 Improved
 ########
-
+- :ref:`SetUncertainties <algm-SetUncertainties-v1>` now provides a "custom" mode, which lets the user arbitrarily replace any error value x with y.
```

Comment:
I don't think this comment reads very well. I'm not sure our users are going to make sense of this when they read the release notes. 

Also, as far as I can tell, the algorithm documentation has not been updated, but should contain some important specification updates such as how truncation (as opposed to rounding) is used in the comparison. I would suggest updating the usage documentation to include new examples in addition to text updates.